keyboards/J80/rules.mk

```diff
@@ -0,0 +1,26 @@
+# MCU name
+MCU = atmega32a
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = bootloadHID
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```

Comment:
Since this PCB has no reset button, I would recommend you set this to `lite`; then you can enter the bootloader by holding down the Escape key while plugging the board in (in case you forgot to map a `RESET` keycode).

Reply:
Set bootmagic to `lite`